### DISCUSSION TITLE
Release supporting aarch64-apple-darwin

### DISCUSSION THREAD

**Discussion by csmulhern** [ispc/ispc#2046] [Q&A] [answered]
It seems like since the last release (1.15.0) there has been support added to build a universal binary for macOS that supports running on aarch64 machines. What's the timeline for a new release (and is there anything I can do to help)?

**Comment by dbabokin** [comment]
Some stuff was added to make it work on M1, but more changes are needed. Some of the changes I have already in a branch, but they are not complete. I hope to get it done till next release, which is going to be some time in the beginning of April.

**Reply by csmulhern** [reply]
Sounds good, thanks!

**Reply by GhisCo** [reply]
Also very much interested in M1 support, any progress update on this? Would a single call to the compiler generate a single universal binary with two distinct sets of targets, or would it require two distinct calls? (Both methods are perfectly fine by me) - Thank you very much!

**Comment by RoyiAvital** [comment]
Is there an official support for this?

**Reply by GhisBntly** [reply, marked as answer]
See changelog for 1.20 on https://github.com/ispc/ispc/releases : "distribution on macOS now includes x86_64, arm64 and Universal Binaries".
Any version (arm64, Intel, or U.B.) of ispc can generate any architecture, but I don't think it can _produce_ universal binaries, you'll need to _lipo_ them together.
